docs/freebsd/README.md

```diff
@@ -2,7 +2,36 @@
 
 ## How to run syzkaller on FreeBSD using qemu
 
-So far the process is tested only on linux/amd64 host. To build Go binaries do:
+So far the process is only tested am amd64 based hosts.
+The host can be running FreeBSD or Linux.
+
+### Setting up a FreeBSD host
+
+Since some tools (`syz-prog2c`, for example) use `clang-format`, you should
+do a buildworld/installworld with having the entry
+```
+WITH_CLANG_EXTRAS="YES"
+```
+in `/etc/src/conf`.
+
+The required dependencies can be installed by
+```
+sudo pkg install bash gcc git gmake go
+```
+Checking out the sources can be done by
+```
+go get -u -d github.com/google/syzkaller/...
+```
+and building the binaries is done by
+```
+cd go/src/github.com/google/syzkaller/
+gmake
+gmake generate
```

Comment:
`gmake generate` is not necessary as we always check-in generated sources
(and if run, it should be run before `make`)

Reply:
Fixed in https://github.com/google/syzkaller/pull/1025/commits/d0c59b46bf62cb1f9cd262d2d73fc40e1b70fc0d.

Thanks for the comments.